stories/forms/labels/input-label.story.js

```diff
@@ -35,6 +35,12 @@ storiesOf('InputLabel', module)
       requiredIndicator={ '*' }
     />
   ))
+  .add('pass DOM props to inner label tag', () => (
```

Comment:
Please follow the convention established in this file and start each story description with "with"